pkgs/servers/http/nginx/generic.nix

```diff
@@ -64,7 +64,23 @@ stdenv.mkDerivation {
 
   preConfigure = (concatMapStringsSep "\n" (mod: mod.preConfigure or "") modules);
 
-  patches = stdenv.lib.optionals (stdenv.hostPlatform != stdenv.buildPlatform) [
+  patches = [
+    # HTTP/2: reject zero length headers with PROTOCOL_ERROR (from 1.16.1)
+      (fetchpatch {
```

Comment:
Thanks a lot, minor nitpick though: Indentation should be 2 spaces, but I merged anyway so we have this in the channel ASAP.